src/pages/UpSertUser/hooks/useGetUserByID.ts

```diff
@@ -0,0 +1,33 @@
+import { gql } from "graphql-request";
+import { useQuery } from "react-query";
+import { GetUserByIdQueryQuery, GetUserByIdQueryQueryVariables } from "src/generated/graphql";
+
+import useQueryClient from "src/hooks/useQueryClient";
+
+const useGetUserByID = (userId: string) => {
+    const queryClient = useQueryClient();
+    const result = useQuery(["GetUserByIDQuery", userId], async () => {
+        const result = await queryClient.request<
+            GetUserByIdQueryQuery,
+            GetUserByIdQueryQueryVariables
+        >(
+            gql`
+                query GetUserByIDQuery($_eq: uuid!) {
+                    users(where: { userId: { _eq: $_eq } }) {
```

Comment:
```suggestion
                    users(where: { userId: { _userId: $_eq } }) {
```

Reply:
![image](https://user-images.githubusercontent.com/58205909/133919322-75cc1d44-11df-4bac-a845-44d8cec0d190.png)

Reply:
I change the query